docs/ui/insight/tools/browser.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Browser
+linkTitle: Browser
+description: Visualize and manage your data structures
+weight: 10
+
+---
+
+**Browser** offers [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) support for lists, hashes, strings, sets, sorted sets, streams, as well as for [RedisJSON](/docs/stack/json).
```

Comment:
Browser is a powerful tool to visualize and interact with the data stored in your Redis and Redis Stack database. 

With the Browser tool, you can explore Redis keys, their values, and metadata, as well as perform CRUD (create, read, update, and delete) operations on strings, hashes, sets, sorted sets, lists, streams, and [RedisJSON](/docs/stack/json).

Navigate through and analyze your list of keys quicker by grouping keys per their namespaces using the Tree view, or use filters per key name or Redis data type to find the keys you need.

The Browser tool also helps you to view, validate, and manage your key values in a human-readable format using formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, ASCII, etc.), as well as automatically decompress your data to display it in a human-readable format, supporting GZIP, LZ4, SNAPPY, and other compressions.

Leverage the [Search](docs/stack/search/) capability to see and work with found keys and their values, or use a dedicated form to create a secondary index.

Delete keys in bulk from your Redis or Redis Stack database based on the filters set in Browser or Upload data from a file in bulk using Bulk Actions.